docs/k0sctl-install.md

```diff
@@ -0,0 +1,171 @@
+
+INFO k0sctl 0.0.0 Copyright 2021, Mirantis Inc.   
+INFO Anonymized telemetry will be sent to Mirantis. 
+INFO By continuing to use k0sctl you agree to these terms: 
+INFO https://k0sproject.io/licenses/eula          
+INFO ==> Running phase: Connect to hosts 
+INFO [ssh] 10.0.0.1:22: connected              
+INFO [ssh] 10.0.0.2:22: connected              
+INFO ==> Running phase: Detect host operating systems 
+INFO [ssh] 10.0.0.1:22: is running Ubuntu 20.10 
+INFO [ssh] 10.0.0.2:22: is running Ubuntu 20.10 
+INFO ==> Running phase: Prepare hosts    
+INFO [ssh] 10.0.0.1:22: installing kubectl     
+INFO ==> Running phase: Gather host facts 
+INFO [ssh] 10.0.0.1:22: discovered bond0 as private interface 
+INFO [ssh] 10.0.0.1:22: discovered 10.12.18.133 as private address 
+INFO ==> Running phase: Validate hosts   
+INFO ==> Running phase: Gather k0s facts 
+INFO ==> Running phase: Download K0s on the hosts 
+INFO [ssh] 10.0.0.2:22: downloading k0s 0.10.0 
+INFO [ssh] 10.0.0.1:22: downloading k0s 0.10.0 
+INFO ==> Running phase: Configure K0s    
+WARN [ssh] 10.0.0.1:22: generating default configuration 
+INFO [ssh] 10.0.0.1:22: validating configuration 
+INFO [ssh] 10.0.0.1:22: configuration was changed 
+INFO ==> Running phase: Initialize K0s Cluster 
+INFO [ssh] 10.0.0.1:22: installing k0s controller 
+INFO [ssh] 10.0.0.1:22: waiting for the k0s service to start 
+INFO [ssh] 10.0.0.1:22: waiting for kubernetes api to respond 
+INFO ==> Running phase: Install workers  
+INFO [ssh] 10.0.0.1:22: generating token       
+INFO [ssh] 10.0.0.2:22: writing join token     
+INFO [ssh] 10.0.0.2:22: installing k0s worker  
+INFO [ssh] 10.0.0.2:22: starting service       
+INFO [ssh] 10.0.0.2:22: waiting for node to become ready 
+INFO ==> Running phase: Disconnect from hosts 
+INFO ==> Finished in 2m2s                
+INFO k0s cluster version 0.10.0 is now installed  
+INFO Tip: To access the cluster you can now fetch the admin kubeconfig using: 
+INFO      k0sctl kubeconfig              
+```
+
+And, -- presto your k0s cluster is up and running
+
+Get kubeconfig:
+```
+$ k0sctl kubeconfig > kubeconfig
+$ KUBECONFIG=kubeconfig kubectl get pods -A
+NAMESPACE     NAME                                       READY   STATUS    RESTARTS   AGE
+kube-system   calico-kube-controllers-5f6546844f-w8x27   1/1     Running   0          3m50s
+kube-system   calico-node-vd7lx                          1/1     Running   0          3m44s
+kube-system   coredns-5c98d7d4d8-tmrwv                   1/1     Running   0          4m10s
+kube-system   konnectivity-agent-d9xv2                   1/1     Running   0          3m31s
+kube-system   kube-proxy-xp9r9                           1/1     Running   0          4m4s
+kube-system   metrics-server-6fbcd86f7b-5frtn            1/1     Running   0          3m51s
+```
+
+### Upgrade k0s version on your setup
+
+_NOTE: Upgrade will only work if you previously used k0sctl v0.4.0 and you are using v0.5.0_
```

Comment:
It can also work if you installed k0s 0.10 manually. I think this note can be removed.